build-tools/create-pkg/scripts/preinstall

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+DESTINATION=/Library/Frameworks/Xamarin.Android.framework
+
+# Ensure installations of previous versions are cleaned up 
+# by deleting all content in our installation directory root.
+if [ -d "$DESTINATION" ]; then
+	rm -rf "$DESTINATION"
```

Comment:
Could this be a tad *drastic*?  It means if I do:

```bash
$ cd /Library/Frameworks/Xamarin.Android.framework/Versions
$ cp -R 9.2.0-2 9.2.0-2.bk
```

My "backup" directory will *also* be nuked.

What does iOS do here?

Reply:
This has been updated to only remove the previously installed version as suggested.